### DISCUSSION TITLE
Is brier score available for survival svm?

### DISCUSSION THREAD

**Discussion by xiaojialong0518** [sebp/scikit-survival#176] [Q&A]
Hello,all
I am using brier score to evaluate cox, survival svm,and RSF models. Brier score works on cox and RSF,but not on survival svm because svm doesn't have a "predict_survival_function". I am wondering if brier score can be used to evaluate survival SVM model and how am I supposed to do it. I really appreciate your help, thanks a lot.

Xiao jialong

**Comment by sebp** [comment]
You are correct, survival SVM can only predict a risk score and not a 
survival function, therefore assessing it in terms of calibration, which 
Brier score does, is not possible. The survival SVM can only be assessed 
in terms of discrimination (c-index, time-dependent ROC).

**Reply by xiaojialong0518** [reply]
Thanks a lot. This is really helpful. This package is so great that I abandoned R package. Thanks for your work.

**Reply by sultanoid** [reply]
It really helps. Thanks for your valuable comment.
